hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/KeyManagerImpl.java

```diff
@@ -1749,6 +1766,9 @@ public OzoneFileStatus getFileStatus(OmKeyArgs args) throws IOException {
           volumeName, bucketName, keyName);
       OmKeyInfo fileKeyInfo = metadataManager.getKeyTable().get(fileKeyBytes);
       if (fileKeyInfo != null) {
+        if (args.getRefreshPipeline()) {
+          refreshPipeline(fileKeyInfo);
```

Comment:
For renameKey this is not the code being used. OMKeyRenameRequest is the code used.
And what is the use case to update it here?